docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,330 @@ sidebar_position: 2
+    name: ${pgo_cluster_name}
+    pg-cluster: ${pgo_cluster_name}
+    pgo-version: 4.7.0
+    pgouser: admin
+  name: ${pgo_cluster_name}
+  namespace: ${cluster_namespace}
+spec:
+  BackrestStorage:
+    accessmode: ReadWriteMany
+    matchLabels: ""
+    name: ""
+    size: 1G
+    storageclass: ""
+    storagetype: create
+    supplementalgroups: ""
+  PrimaryStorage:
+    accessmode: ReadWriteMany
+    matchLabels: ""
+    name: ${pgo_cluster_name}
+    size: 1G
+    storageclass: ""
+    storagetype: create
+    supplementalgroups: ""
+  ReplicaStorage:
+    accessmode: ReadWriteMany
+    matchLabels: ""
+    name: ""
+    size: 1G
+    storageclass: ""
+    storagetype: create
+    supplementalgroups: ""
+  annotations: {}
+  ccpimage: crunchy-postgres-ha
+  ccpimageprefix: ${cluster_image_prefix}
+  ccpimagetag: centos8-13.3-4.7.0
+  clustername: ${pgo_cluster_name}
+  database: ${pgo_cluster_name}
+  exporterport: "9187"
+  limits: {}
+  name: ${pgo_cluster_name}
+  pgDataSource:
+    restoreFrom: ""
+    restoreOpts: ""
+  pgbadgerport: "10000"
+  pgoimageprefix: ${cluster_image_prefix}
+  podAntiAffinity:
+    default: preferred
+    pgBackRest: preferred
+    pgBouncer: preferred
+  port: "5432"
+  tolerations: []
+  user: hippo
+  userlabels:
+    pgo-version: 4.7.0
+EOF
+
+kubectl apply -f "${pgo_cluster_name}-pgcluster.yaml"
+```
+
+Ensure all the pods in `my-postgresql` is running (it will take few minutes):
+
+```
+kubectl get pod -n my-postgresql
+```
+
+## Verifying PostgreSQL Connection and Create Schema
+
+To verify the PostgreSQL connection:
+
+```
+# name of the user
+export pgo_cluster_username=hippo
+
+# get the password of the user and set it to a recognized psql environmental variable
+export PGPASSWORD=$(kubectl -n "${cluster_namespace}" get secrets \
+  "${pgo_cluster_name}-${pgo_cluster_username}-secret" -o "jsonpath={.data['password']}" | base64 -d)
+
+# set up a port-forward either in a new terminal, or in the same terminal in the background:
+kubectl -n ${cluster_namespace} port-forward svc/hippo 5432:5432 &
+
+psql -h localhost -U "${pgo_cluster_username}" "${pgo_cluster_name}"
+```
+
+Download the schema and sample data load it to the database:
+
+```
+curl -LO https://raw.githubusercontent.com/spring-petclinic/spring-petclinic-rest/master/src/main/resources/db/postgresql/initDB.sql
+
+psql -h localhost -U "${pgo_cluster_username}" "${pgo_cluster_name}" -f initDB.sql
+
+curl -LO https://raw.githubusercontent.com/spring-petclinic/spring-petclinic-rest/master/src/main/resources/db/postgresql/populateDB.sql
+
+psql -h localhost -U "${pgo_cluster_username}" "${pgo_cluster_name}" -f populateDB.sql
+```
+
+## Building the PetClinic App
+
+The application should be prepared to use the bindings projected into the
+container.  This section explains how to use [Cloud Native
+Buildpacks](https://buildpacks.io) and [Spring Cloud
+Bindings](https://github.com/spring-cloud/spring-cloud-bindings) Java library to
```

Comment:
I have no idea why we need the "spring-cloud-bindings". As a reader, seeing all the dependencies listed in the sample, I'd be worry about the complexity of ServiceBinding. 
For example: if I'm a nodejs developer, is there are "nodejs-cloud-binding" library needed?
We should aim for something simpler. 

Can't we do it without spring-cloud-binding?

Reply:
I had a discussion with Predrag today. Decided to remove this section and provide a published OCI image of the application.